library/psa_crypto_client.c

```diff
@@ -19,7 +19,6 @@
  */
 
 #include "common.h"
-#include "psa_crypto_service_integration.h"
```

Comment:
Isn't it going to be missing here ?

Reply:
Is it actually necessary here? This file is client-only. `PSA_CRYPTO_SPM` is for the secure side only.

Reply:
Currently we have MBEDTLS_PSA_CRYPTO_C enabled => MBEDTLS_PSA_CRYPTO_CLIENT enabled
and I don't see how TFM could do without having their own implementation of at least psa_set_key_domain_parameters() thus I am not sure how this could work without including `psa_crypto_service_integration.h` here.